lib/platform/gitlab/http.ts

```diff
@@ -1,4 +1,6 @@
+import { logger } from '../../logger';
 import { GitlabHttp } from '../../util/http/gitlab';
+import { GitlabUserStatus } from './types';
```

Comment:
```suggestion
import type { GitlabUserStatus } from './types';
```

Reply:
Why is that necessary?

Reply:
This will force import removal from final is file. So we don't import empty type files at runtime.